app/src/ui/open-pull-request/open-pull-request-dialog.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react'
+import { IPullRequestState } from '../../lib/app-state'
+import { Repository } from '../../models/repository'
+import { DialogFooter, OkCancelButtonGroup, Dialog } from '../dialog'
+import { Dispatcher } from '../dispatcher'
+
+interface IOpenPullRequestDialogProps {
+  readonly repository: Repository
+  readonly dispatcher: Dispatcher
+  readonly pullRequestState: IPullRequestState
+
+  /** Called to dismiss the dialog */
+  readonly onDismissed: () => void
+}
+
+/** The component for start a pull request. */
+export class OpenPullRequestDialog extends React.Component<IOpenPullRequestDialogProps> {
+  private onCreatePullRequest = () => {
+    this.props.dispatcher.createPullRequest(this.props.repository)
+    // TODO: create pr from dialog pr stat?
+    this.props.dispatcher.recordCreatePullRequest()
+  }
+
+  private renderHeader() {}
+
+  private renderContent() {}
+
+  private renderFooter() {
+    return (
+      <DialogFooter>
+        <OkCancelButtonGroup
+          okButtonText="Create Pull Request"
+          okButtonTitle="Create pull request on GitHub."
```

Comment:
No need to do it now, but of course here we might need to specify if it's on GitHub, or on a GHES/whatever instance maybe?

Reply:
Good point - Also wasn't 100% sure we wanted this or not - I was just thinking of ways to indicate that this would jump out to browser.. also thought about putting an icon on this button to indicate that.